tests/test_init.py

```diff
@@ -2,10 +2,23 @@
 import click
 from click.testing import CliRunner
 from egasub.cli import init
+from egasub.cli import main
+import yaml
 
 def test_init_function():    
     runner = CliRunner()
-    
     with runner.isolated_filesystem():
-        result = runner.invoke(init)
-        assert True
\ No newline at end of file
+        result = runner.invoke(main,['init','--ega_submitter_account','test_account','--ega_submitter_password','test_password','--icgc_id_service_token','test_token','--icgc_project_code','test_code'])
```

Comment:
it's probably better to directly test the `init_workspace` function, instead of launching it from click cli. This way we don't need to use command options to collect those values, we can just use what you used before, the `prompt` function imported from click